servers_v7.json

```
@@ -108,4 +108,8 @@
     "name": "Eradicationdustry",
     "address": ["n1.yeet.ml:6577"]
   }
```

```suggestion
  },
```
The comma there too :)